test/data/certs/generate-test-certs

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -xe
```

Comment:
(super nitpick) any reason to keep the `-x` here? I don't mind much but it seems fine to (to me) to remove it